### PR TITLE
Move all deps to devDeps

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "An experimental PopcornTime client",
   "main": "main.js",
   "scripts": {
-    "build": "npm run build-main & npm run build-renderer",
+    "build": "npm run build-main && npm run build-renderer",
```
Comment (thangngoc89) [inline package.json]: They can run in parallel right ?

**Comment by thangngoc89** [comment]
You should run `npm install -D lodash && npm uninstall -D lodash`. npm will sort the dependencies list based on alphabet

**Comment by amilajack** [comment]
On windows this was causing issues. The gulpfile that failed with eslint validation on travis passed on windows. For now lets run them serially.

File: package.json [hunk 2]
```diff
@@ -130,14 +129,12 @@
...
     "webpack-hot-middleware": "^2.12.1",
-    "why-did-you-update": "^0.0.8"
-  },
-  "dependencies": {
+    "why-did-you-update": "^0.0.8",
     "babel-runtime": "^6.9.2",
     "classnames": "^2.2.5",
     "css-modules-require-hook": "^4.0.1",
     "electron-debug": "^1.0.1",
-    "electron-prebuilt": "^1.2.8",
```
Comment (thangngoc89) [inline package.json]: There is no different with this change. This is semver patch

**Comment by thangngoc89** [comment]
How about this ? 
 https://github.com/thangngoc89/popcorn-time-desktop/tree/parallel-webpack

Run webpack builds on Windows is extremely slow.

**Comment by amilajack** [comment]
parallel-webpack looks pretty nice! PR?

**Comment by amilajack** [comment]
And my spectron tests are failing on my local machine so i downgraded the electron version temporarily

**Comment by thangngoc89** [comment]
> And my spectron tests are failing on my local machine so i downgraded the electron version temporarily

It does change anything. If you want to downgrade it, define it with no `^` in the begining (fixed version)

**Comment by amilajack** [comment]
Oh right. Patch version.

**Comment by amilajack** [comment]
Not sure how the builds are passing on travis but not on my local machine. hhm

**Pull request merged**